src/components/new-request/search.vue

```diff
@@ -611,6 +627,15 @@ export default class Search extends Mixins(CommonMixin) {
   onRequestActionChange (request: RequestActionsI): void {
     this.request = request
 
+    // calculate whether to show tooltip
+    // (in next tick after DOM update)
+    Vue.nextTick(() => {
+      const el = document.querySelector('.request-action-select .v-select__selection') as any
+      const offsetWidth = el?.offsetWidth as number
+      const scrollWidth = el?.scrollWidth as number
+      this.showRequestActionTooltip = (offsetWidth < scrollWidth)
```

Comment:
This is a pretty cool solution.

Reply:
It's also the _only_ solution, because the text clipping is done automatically by the browser.